addons/payment_stripe/models/payment_acquirer.py

```diff
@@ -108,7 +108,13 @@ def action_stripe_connect_account(self, menu_id=None):
             connected_account = self._stripe_fetch_or_create_connected_account()
 
             # Link generation
-            menu_id = menu_id or self.env.ref('payment.payment_acquirer_menu').id
+            if not menu_id:
+                # Fall back on account payment menu
+                menu = self.env.ref('account_payment.payment_acquirer_menu', False)
+                # If account_payment is not installed, fall back on the technical menu in payment
+                menu = menu or self.env.ref('payment.payment_acquirer_menu', False)
+                menu_id = menu.id
```

Comment:
Note for myself: Test without acc_pay menu & verify redirection link lands on base /web page.